app/views/patients/_form_history.html.slim

```diff
@@ -0,0 +1,49 @@
+.row
+  .medium-4.columns
+    = link_to 'Add New Annual Evaluation', new_patient_annual_evaluation_path(@patient), { class: "button secondary small expand" }
+    table.table
+      tr
+        th Previous Annual Evaluations
+      - for episode in @patient.annual_evaluations do
+        tr class="clickable-row" data-href=edit_polymorphic_url([@patient, episode])
```

Comment:
You can use edit_annual_evluations_url, etc. all over now since you know the type. can you get rid of all the polymorphic_url bits?